deploy/charts/approver-policy/templates/deployment.yaml

```diff
@@ -51,18 +55,28 @@ spec:
           - --webhook-ca-secret-namespace={{.Release.Namespace}}
           - --webhook-certificate-dir={{.Values.app.webhook.certificateDir}}
 
-        {{- if .Values.volumeMounts }}
         volumeMounts:
-{{ toYaml .Values.volumeMounts | indent 10 }}
+        {{- with .Values.volumeMounts }}
+        {{- toYaml . | nindent 10 }}
         {{- end }}
+        - mountPath: /tmp
+          name: temp-dir
 
         resources:
           {{- toYaml .Values.resources | nindent 10 }}
 
-      {{- if .Values.volumes }}
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities: { drop: ["ALL"] }
+          readOnlyRootFilesystem: true
```

Comment:
These improvements seem to be unrelated to this PR. Not a big deal though.

Reply:
I had to add these improvements in order for the kyverno test to not fail.